plugins/modules/nxos_user.py

```diff
@@ -407,7 +392,12 @@ def main():
 
     if commands:
         if not module.check_mode:
-            load_config(module, commands)
+            responses = load_config(module, commands)
+            for resp in responses:
+                if resp.lower().startswith("warning"):
+                    result["warnings"].append(resp)
```

Comment:
This would presumably result in something like `WARNING: Warning something something` in the log right? Should we strip the "warning" prefix from response here?

For that matter, do you have something to trigger this branch? We should probably have a test that hits this.

Reply:
Yeah, that's right. I thought the same, but didn't want to mess with the actual message received from the device. On second thoughts, this may not be a bad idea. I'll change this and add a test.